tests/testthat/test-lancelot-vaccination.R

```diff
@@ -3220,3 +3220,222 @@ test_that("run sensible vaccination schedule with waning and boosters", {
   phase4 <- 150:200
   expect_false(all(y$cum_n_S_vaccinated[, 4, , phase4] == 0))
 })
+
+test_that("Can validate vaccine skip move inputs", {
+  expect_error(lancelot_parameters(1, "london", vacc_skip_from = 2),
+               "n_vacc_classes = 1 so require vacc_skip_from = 1")
+
+  expect_error(lancelot_parameters(1, "london", vacc_skip_to = 3),
+               "n_vacc_classes = 1 so require vacc_skip_to = 1")
+
+  expect_error(lancelot_parameters(1, "london",
+                                   vacc_skip_progression_rate = c(1, 2)),
+               "vacc_skip_progression_rate must be a scalar or a")
+
+  expect_error(lancelot_parameters(1, "london", rel_susceptibility = c(1, 1, 1),
+                                   vacc_skip_progression_rate = rep(0, 19),
+                                   vacc_skip_from = 3L, vacc_skip_to = 2L),
+               "Require vacc_skip_from <= vacc_skip_to")
+
+  expect_error(lancelot_parameters(1, "london", rel_susceptibility = c(1, 1, 1),
+                                   vacc_skip_from = 1L, vacc_skip_to = 3L,
+                                   vacc_skip_weight = 1.4),
+               "'vacc_skip_weight' must lie in")
+
+  expect_error(lancelot_parameters(1, "london", rel_susceptibility = c(1, 1, 1),
+                                   vacc_skip_from = 1L, vacc_skip_to = 3L,
+                                   vacc_skip_weight = -1),
+               "'vacc_skip_weight' must lie in")
+  expect_error(lancelot_parameters(1, "london", rel_susceptibility = c(1, 1, 1),
```

Comment:
I think you've got the exact same test repeated twice?

Reply:
Removed, thanks